src/app/busca-ativa/diabeticos/Diabetes.tsx

```
@@ -27,7 +34,7 @@ export const Diabetes : React.FC<DiabetesProps> = ({
     const router = useRouter();
     const path = usePathname();
     const visao = session?.user.perfis.includes(5) || session?.user.perfis.includes(8) ? "aps" : "equipe"
-    useEffect(() => {router.push(`${path}?visao=${visao}`)}, [visao]);
+    useEffect(() => {router.push(`${path}?aba=${''}&sub_aba=${''}&visao=${visao}`)}, [visao]);
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: path

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)